website/landingpage/dev/src/partials/features.html

```diff
@@ -0,0 +1,140 @@
+
+  <section class="is-centered">
```

Comment:
You can remove `<section>` here and move `is-centered` class to the `row`

Reply:
done